packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -1180,6 +1180,57 @@ void main() {
     }
   }, variant: TargetPlatformVariant.all());
 
+  testWidgetsWithLeakTracking('The onSelected gets called only when a selection is made in a nested menu',
```

Comment:
Could you add a test (with `expect(tester.takeException(), isNull)`) to verify that the exception of the disposed controller does not happen anymore?

Reply:
great suggestion, didn't know about the takeException method. Very handy.